packages/commonwealth/server/routes/bulkOffchain.ts

```diff
@@ -35,48 +31,41 @@ const bulkOffchain = async (
   const { community } = req;
 
   // parallelized queries
-  const [adminsAndMods, numVotingThreads, numTotalThreads, communityBanner] =
-    await (<
-      Promise<[AddressInstance[], number, number, CommunityBannerInstance]>
-    >Promise.all([
-      // admins
-      models.Address.findAll({
-        where: {
-          // @ts-expect-error StrictNullChecks
-          community_id: community.id,
-          [Op.or]: [{ role: 'admin' }, { role: 'moderator' }],
-        },
-        attributes: ['address', 'role'],
-      }),
-      models.Thread.count({
-        where: {
-          // @ts-expect-error StrictNullChecks
-          community_id: community.id,
-          stage: 'voting',
-        },
-      }),
-      models.Thread.count({
-        where: {
-          // @ts-expect-error StrictNullChecks
-          community_id: community.id,
-          marked_as_spam_at: null,
-        },
-      }),
-      models.CommunityBanner.findOne({
-        where: {
-          // @ts-expect-error StrictNullChecks
-          community_id: community.id,
-        },
-      }),
-    ]));
+  const [adminsAndMods, numVotingThreads, numTotalThreads] = await (<
+    Promise<[AddressInstance[], number, number]>
+  >Promise.all([
+    // admins
+    models.Address.findAll({
+      where: {
+        // @ts-expect-error StrictNullChecks
+        community_id: community.id,
+        [Op.or]: [{ role: 'admin' }, { role: 'moderator' }],
+      },
+      attributes: ['address', 'role'],
+    }),
+    models.Thread.count({
+      where: {
+        // @ts-expect-error StrictNullChecks
+        community_id: community.id,
+        stage: 'voting',
+      },
+    }),
+    models.Thread.count({
```

Comment:
We should combine these counts into a single query using either `SUM` or `COUNT`

```
SELECT
    SUM(CASE WHEN stage = 'voting' THEN 1 ELSE 0 END),
    SUM(CASE WHEN marked_as_spam_at IS NULL THEN 1 ELSE 0 END)
FROM "Threads"
WHERE community_id = 'stargatetoken';

SELECT
    COUNT(CASE WHEN stage = 'voting' THEN 'voting' ELSE NULL END),
    COUNT(CASE WHEN marked_as_spam_at IS NULL THEN True ELSE NULL END)
FROM "Threads"
WHERE community_id = 'stargatetoken';
```

The queries above have basically the same query plan as a single count query since both iterate through all threads with `community_id` given the lack of indexes on `marked_as_spam_at` and `stage`.

Reply:
We have a ticket to remove this route completely -> https://github.com/hicommonwealth/commonwealth/issues/8995 -- so I think this update can be skipped